prove/Develop05/CheckListGoal.cs

```diff
@@ -45,11 +52,12 @@ public void set(string name, string description, int points, string goalType, in
     }
       public override void ShowListGoal()
     {
-        Console.WriteLine($"{_name} ({_description})--- Currently completed: {_tries}/{_timesToGetBonus}");
-        if (_tries == _timesToGetBonus)
-        {
-          _tries = 0;
-        }
+      if (_tries == _timesToGetBonus)
+      {
+        _tries = 0;
+      }
+      Console.WriteLine($"{_name} ({_description})--- Currently completed: {_tries}/{_timesToGetBonus}");
+        
```

Comment:
I think the 57 line is unnecessary because I already assigned the value of zero to _tries but I just wanted to ensure that the message was correct.